Competition/src/main/java/competition/subsystems/pose/PoseSubsystem.java

```diff
@@ -21,15 +22,25 @@
     DriveSubsystem drive;
 
     public enum FieldLandmark {
-        HabLevelOne,
-        RightLoadingStation,
-        CargoShipSix
+        CargoShipRightThree,
+        CargoShipFrontRight,
+        HabRightTwo,
+        HabRightOne,
+        HabTouchingRamp,
+        PanelLoadRight,
+        RocketFrontRight,
+        
     }
 
     private Map<FieldLandmark, FieldPosePropertyManager> landmarkToLocation;
-    private final FieldPosePropertyManager habLevelOneManager;
-    private final FieldPosePropertyManager rightLoadingStationManager;
-    private final FieldPosePropertyManager cargoShipSixManager;
+    public final FieldPosePropertyManager cargoShipRightThreeManager;
```

Comment:
instead of making these guys public, looks like John had created a `getFieldPoseForLandmark(landmark)` method that you can call from the OperatorCommandMap. See `SetPoseToFieldLandmarkCommand` for an example usage.